### PR TITLE
update conatcts payload

### PR DESCRIPTION
**Pull request by Qwertic**
### Description
Adding info about the updates in: 
- `Contact` payload showing `customFields`
- `POST messages/send` endpoint applies the 24h policy check

Regarding the 24h policy check, we now don't allow sending messages outside of the 24h range. Templates must be used for it

#### Send message warning
Screenshot 2023-02-10 at 10 51 49

#### Updated contact payload
Screenshot 2023-02-10 at 12 49 52

### CODE REVIEW

File: docs/api_reference/messages_api/post_send_messages.md [hunk 1]
```diff
@@ -4,6 +4,10 @@ title: POST /messages/send
 
 # POST /messages/send
 
+:::caution
+After 24h without a reply from the customer, it is not possible to send regular messages, you'll need to use a **Template** message, see examples below.
```
Comment (proudlygeek) [inline docs/api_reference/messages_api/post_send_messages.md]:
```suggestion
After 24 hours without a response from the customer, it is not possible to send regular messages. You will need to use a **template message**, as shown in the examples below.
```

File: docs/api_reference/object_types/contact.md [hunk 2]
```diff
@@ -5,13 +5,14 @@ sidebar_position: 1
...
-| `name`        | string | Name of contact                               |
-| `phoneNumber` | string | Phone number of the contact                   |
-| `avatarUrl`   | string | URL of the user avatar                        |
-| `createdAt`   | string | Date of contact creation (ISO 8601 formatted) |
-| `source`      | Source | Source of conversation                        |
-| `href`        | string | Link to dashboard CRM contact                 |
-| `tags`        | string | List of associated tags                       |
+| Field          | Type   | Description                                   |
+| :------------- | :----- | :-------------------------------------------- |
+| `uuid`         | string | An unique contact ID generated by Callbell    |
```
Comment (proudlygeek) [inline docs/api_reference/object_types/contact.md]:
```suggestion
| `uuid`         | string | The unique contact ID generated by Callbell    |
```

**Review by proudlygeek — APPROVED** [review]
LGTM 👍 just a minor suggestion

File: docs/api_reference/messages_api/post_send_messages.md (hunk 1 above)
Comment (proudlygeek) [reply docs/api_reference/messages_api/post_send_messages.md]: Also we could link the "example" word to the paragraph where we show how to send templates?

Comment (Qwertic) [reply docs/api_reference/messages_api/post_send_messages.md]: ah yes 🆗

**Pull request merged**